*enclosing-call-site.js:16
      throw new Error('an error!')
                ^

Error: an error!
    at functionD (*enclosing-call-site-min.js:1:156)
        -> (*enclosing-call-site.js:16:17)
    at functionC (*enclosing-call-site-min.js:1:97)
        -> (*enclosing-call-site.js:10:3)
    at functionB (*enclosing-call-site-min.js:1:60)
        -> (*enclosing-call-site.js:6:3)
    at functionA (*enclosing-call-site-min.js:1:26)
        -> (*enclosing-call-site.js:2:3)
    at Object.<anonymous> (*enclosing-call-site-min.js:1:199)
        -> (*enclosing-call-site.js:24:3)
    at Module._compile (internal/modules/cjs/loader.js:*)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:*)
    at Module.load (internal/modules/cjs/loader.js:*)
    at Function.Module._load (internal/modules/cjs/loader.js:*)
    at Module.require (internal/modules/cjs/loader.js:*)
